src/main/java/mate/jdbc/repository/impl/RepositoryManufacturer.java

```diff
@@ -0,0 +1,130 @@
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        try (Connection connection = ConnectionSupplier.getConnection();
+                PreparedStatement insertStatement = connection.prepareStatement(
+                        "SELECT * "
+                                + "FROM manufacturers "
+                                + "WHERE id = ? "
+                                + "AND is_deleted IS FALSE;")) {
+            insertStatement.setLong(1, id);
+            insertStatement.executeQuery();
+            return parseResult(insertStatement.getResultSet()).stream().findFirst();
+        } catch (SQLException e) {
+            throw new DataProcessingException(
+                    CONNECTION_EXCEPTION + RepositoryManufacturer.class.getName(),
+                    e);
+        }
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        try (Connection connection = ConnectionSupplier.getConnection();
+                Statement getAllStatement = connection.createStatement()) {
+            ResultSet resultSet = getAllStatement.executeQuery(
+                    "SELECT * "
+                            + "FROM manufacturers "
+                            + "WHERE is_deleted IS FALSE;");
+            return parseResult(resultSet);
+        } catch (SQLException e) {
+            throw new DataProcessingException(
+                    CONNECTION_EXCEPTION + RepositoryManufacturer.class.getName(),
+                    e);
+        }
+    }
+
+    @Override
+    public Optional<Manufacturer> update(Manufacturer manufacturer) {
+        try (Connection connection = ConnectionSupplier.getConnection();
+                PreparedStatement insertStatement = connection.prepareStatement(
+                        "UPDATE manufacturers "
+                                + "SET " + COLUMN_NAME + " = ?, " + COLUMN_COUNTRY + " = ? "
+                                + "WHERE id = ? " + "AND is_deleted IS FALSE;")) {
+            insertStatement.setString(1, manufacturer.getName());
+            insertStatement.setString(2, manufacturer.getCountry());
+            insertStatement.setLong(3, manufacturer.getId());
+            insertStatement.executeUpdate();
+            return get(manufacturer.getId());
+        } catch (SQLException e) {
+            throw new DataProcessingException(
+                    CONNECTION_EXCEPTION + RepositoryManufacturer.class.getName(),
+                    e);
+        }
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        try (Connection connection = ConnectionSupplier.getConnection();
+                PreparedStatement insertStatement = connection.prepareStatement(
+                        "UPDATE manufacturers "
+                                + "SET is_deleted = true "
+                                + "WHERE id = ? "
+                                + "AND is_deleted IS FALSE;")) {
+            insertStatement.setLong(1, id);
+            return insertStatement.execute();
```

Comment:
```suggestion
            return insertStatement.executeUpdate() > 0;
```
As stated in common mistakes